elfloader-tool/src/arch-riscv/crt0.S

```diff
@@ -54,10 +54,25 @@ _start:
   mv s0, a0 /* preserve a0 (hart id) in s0 */
   mv s2, a1 /* preserve a1 (dtb) in s2 */
 
-#ifdef CONFIG_IMAGE_BINARY
+  /* Set a stack so that we can call into C functions */
+  la sp, (elfloader_stack_alloc + BIT(12))
```

Comment:
seem this als addresses https://github.com/seL4/seL4_tools/issues/84 then.

Reply:
I made https://github.com/seL4/seL4_tools/pull/136 to explicitly fix the early boot issues.